snykTask/src/__tests__/test-severity-threshold.ts

```diff
@@ -0,0 +1,21 @@
+import { isNotValidThreshold, Severity } from "../index";
+
+describe("isNotValidThreshold", () => {
+  const validSeverityThresholds = [
+    Severity.CRITICAL,
+    Severity.HIGH,
+    Severity.MEDIUM,
+    Severity.LOW
+  ];
+  test("returns true if invalid severity threshold", () => {
+    const isInvalid = isNotValidThreshold("hey");
+    expect(isInvalid).toBe(true);
+  });
+  test("returns false for all valid severity thresholds", () => {
```

Comment:
**suggestion (non-blocking):** Use jest's `each` for easier reading of test output

https://github.com/snyk/snyk-azure-pipelines-task/pull/101/files

Reply:
Very nice idea, thanks  - I've incorporated 😃